inc/html.class.php

```diff
@@ -1010,10 +997,8 @@ static function progressBar($id, array $options = []) {
    /**
     * Create a Dynamic Progress Bar
     *
-    * @param $msg initial message (under the bar) (default '&nbsp;')
-    *
-    * @return nothing
-    **/
+    * @param string $msg initial message (under the bar) (default '&nbsp;')
+    */
```

Comment:
It returns void.

Reply:
this uses the same "output" from the previous comment